x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/rolemapping/GetRoleMappingsAction.java

```diff
@@ -5,14 +5,14 @@
  */
 package org.elasticsearch.xpack.core.security.action.rolemapping;
 
-import org.elasticsearch.action.Action;
+import org.elasticsearch.action.GenericAction;
 
 /**
- * Action to retrieve one or more role-mappings from X-Pack security
+ * GenericAction to retrieve one or more role-mappings from X-Pack security
```

Comment:
I think the old comment was fine.

Can you remove the extra new line below this line while you are here?